src/entity/MarketingCta.ts

```diff
@@ -23,6 +23,9 @@ export class MarketingCta {
   @Column({ type: 'text' })
   variant: MarketingCtaVariant;
 
+  @Column({ type: 'boolean', default: false })
+  disabled: boolean;
```

Comment:
Maybe for future proofing is `status` with enum better? eg. `active | disabled`

Reply:
Refactored to use enum instead